tests/performance_tests/algorithm_performance_tests_backdeployment.py

```diff
@@ -121,7 +121,7 @@
      "config": {
       "SupplyDrivenDeploymentInst": {
        "calc_method": "ma",
-       "commodities": {"val": "spentfuel_sink_100000"},
+       "commodities": {"val": "spentfuel_sink_1000000"},
```

Comment:
I know I'm petty - I also tested it this works as well
```suggestion
       "commodities": {"val": "spentfuel_sink_1e6"},
```